assets/app/views/builds.html

```diff
@@ -143,11 +141,15 @@
             </dl>
           </div>
         </div>
-        <div class="well" style="margin-bottom: 10px;" ng-repeat="build in buildsByBuildConfig[buildConfigName] | orderObjectsByDate : true">
+        <div class="well build-log" ng-repeat="build in buildsByBuildConfig[buildConfigName] | orderObjectsByDate : true">
```

Comment:
I'd suggest a different class name, maybe just `build`. Eventually we want to surface the actual build log in the UI, which is something else.